CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ All notable changes to this project will be documented in this file. Take a look
 
 <!-- ## [Unreleased] -->
 
+## [3.0.3]
+
+### Fixed
+
+* Fixed `IllegalArgumentException` when trying to decrypt the end of a [CbcLcpResource].
+
+
+#### LCP
+
```

Comment:
It's missing at the bottom:

```
[3.0.3]: https://github.com/readium/kotlin-toolkit/compare/3.0.2...3.0.3
```

Otherwise the link in the heading doesn't work.